src/createAppContainer.js

```diff
@@ -20,8 +20,16 @@ function validateProps(props) {
   if (isStateful(props)) {
     return;
   }
-  // eslint-disable-next-line no-unused-vars
-  const { navigation, screenProps, ...containerProps } = props;
+  /* eslint-disable no-unused-vars */
+  const {
+    navigation,
+    screenProps,
+    persistNavigationState,
```

Comment:
I'm not sure if this is a good thing, but if you keep the old diff, tests will fail